### DISCUSSION TITLE
Cron Job Extension issue

### DISCUSSION THREAD

**Discussion by MaximeBiloa** [supabase/supabase#27912] [Questions] [answered]
Hi, i'm facing an issue when i want to manage job in my project.
I'm trying to schedule new job and i'm this message "permission denied for table job"

**Comment by GaryAustin1** [comment]
You need to provide more info.
Is this a recent instance?
What Postgres role are you using to create the new job?
Maybe show your code...

**Reply by MaximeBiloa** [reply]
My project is quite old and I'm trying to do it in the dashboard with the postgres role.

**Reply by MaximeBiloa** [reply]
I should point out that everything worked fine before. Now I can't program or delete jobs.

**Reply by GaryAustin1** [reply, marked as answer]
Try and remove and re install the extension in the extension UI.  The main postgres role has changed about a year ago.
I see your new message and it is not clear what "before" time frame is.  If it worked yesterday and not today that is different than months ago.

If remove/install does not resolve you will need to provide more info on your code setting things up or contact support (especially if an older instance).

**Reply by MaximeBiloa** [reply]
It works now. Thank you very much.
